ellama.el

```diff
@@ -176,11 +180,30 @@ ARGS contains keys for fine control.
 :buffer BUFFER -- BUFFER is the buffer (or `buffer-name') to insert ellama reply
 in.  Default value is (current-buffer).
 
-:point POINT -- POINT is the point in buffer to insert ellama reaply at."
+:point POINT -- POINT is the point in buffer to insert ellama reply at.
+
+:filter FILTER -- FILTER is a function applied to (partial) response strings.
```

Comment:
note: I'm not sure if this is correct given that we're matching on a potentially partial string. But it matches the current behavior.

Reply:
Ah, nvm. I see that it's partial starting at the beginning, not a fragment.